src/components/BookItem.js

```diff
@@ -1,15 +1,17 @@
-/* eslint-disable no-unused-vars */
 /* eslint-disable react/destructuring-assignment */
 /* eslint-disable react/prop-types */
 import React from 'react';
 import { useDispatch } from 'react-redux';
-import { removeBook } from '../redux/books/books';
+import { deleteBooks } from '../redux/books/books';
+
+const apiKkey = 'kqkewfRoZMkq2lXZaEMG';
+export const baseUrls = `https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/${apiKkey}/books`;
 
 const BookItem = (props) => {
   const dispatch = useDispatch();
+
   const handleRemoveItem = (id) => {
-    dispatch(removeBook(id));
-    console.log('removed book', id);
```

Comment:
While you did well working on some of the project requirements I notice you disabled eslint. It is against microverse policy to disable linters for linters help us write clean code. so it would be nice if you enabled eslint in this file.